tests/beaker_tests/cisco_bgp/test_bgp_defaults.rb

```diff
@@ -135,6 +135,12 @@
   # Default VRF Test Cases
   # ----------------------
 
+  # delete properties not support on platform
+  device = platform
```

Comment:
Perhaps out of scope for this PR but this is going to be painful to merge into feature/grpc as there we are using 'platform' to mean Nexus vs XR, not N7k vs N3k. `platform` as used in this context should really be `product_id` or `product_grouping`.